nixnet/database/database.py

```diff
@@ -69,20 +107,55 @@ def close(self, close_all_refs=False):
 
     def save(self, db_filepath=""):
         # type: (typing.Text) -> None
+        """Saves the open database to a FIBEX 3.1.0 file.
+
+        The file extension must be .xml. If the target file exists, it is overwritten.
+
+        XNET saves to the FIBEX file only features that XNET sessions use to communicate on the network.
+        If the original file was created using non-XNET software,
+        the target file may be missing details from the original file.
+        For example, NI-XNET supports only linear scaling.
+        If the original FIBEX file used a rational equation that cannot be expressed as a linear scaling,
+        XNET converts this to a linear scaling with factor 1.0 and offset 0.0.
+
+        If ``db_filepath`` is empty, the file is saved to the same FIBEX file specified when opened.
+        If opened as a file path, it uses that file path.
+        If opened as an alias, it uses the file path registered for that alias.
```

Comment:
As discussed in-person we should add any docstrings about references to RT anyway, even if we're not supporting RT right now so we don't have to come back and write those comments.